src/Components/index.tsx

```diff
@@ -1,3 +1,22 @@
-export { default as Home } from './Home/Home';
-export { default as CreateVideo } from './CreateVideo/CreateVideo';
-export { Player } from './Import/Player';
\ No newline at end of file
+export { CreateVideo } from './CreateVideo/CreateVideo';
+export { default as Import } from "./Import/Import";
+export { ErrorModal, showError } from "./ErrorToast/ErrorToast";
+export { Header } from "./Shared/Header";
+export { isImage, isVideo, trimmedName } from "./utils";
+export { Loading } from './Loading/Loading';
+export {AddMedia} from './AddMedia/AddMedia';
+
+export class Media {
+  filename: string;
+  type: string;
+  element: JSX.Element;
+  constructor(filename: string, type: string, element: JSX.Element) {
```

Comment:
OOI, why did you decide to use a class here?

Reply:
to make it easier while instantiating a Media Object 😅